internal/pkg/redis/client.go

```diff
@@ -250,18 +252,20 @@ func createRedisClient(
 
 func convertToRedisTopicScheme(topic string) string {
 	// Redis Pub/Sub uses "." for separator and "*" for wild cards.
-	// Since we have standardized on the MQTT style scheme or "/" & "#" we need to
+	// Since we have standardized on the MQTT style scheme or "/" & "#" & "+" we need to
 	// convert it to the Redis Pub/Sub scheme.
 	topic = strings.Replace(topic, StandardTopicSeparator, RedisTopicSeparator, -1)
 	topic = strings.Replace(topic, StandardWildcard, RedisWildcard, -1)
+	topic = strings.Replace(topic, SingleLevelWildcard, RedisWildcard, -1)
 
 	return topic
 }
 
 func convertFromRedisTopicScheme(topic string) string {
```

Comment:
Please add unit test for this function with various inputs/outputs

Reply:
Added new unit tests.